spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/AdjustPrioritiesViaUserPrefsTest.java

```diff
@@ -0,0 +1,151 @@
+package edu.umd.cs.findbugs.detect;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.FindBugs2;
+import edu.umd.cs.findbugs.config.UserPreferences;
+
+class AdjustPrioritiesViaUserPrefsTest extends AbstractIntegrationTest {
+
+    //    @Test
```

Comment:
Is there any reason new commented out code is added? It is not completed yet, or fails, or ... ?